src/main.rs

```diff
@@ -83,6 +83,10 @@ fn resolve_url(url: &str) -> Option<String> {
 	easy.follow_location(true).unwrap();
 	easy.useragent("url-bot-rs/0.1").unwrap();
 
+	let mut headers = List::new();
+	headers.append("Accept-Language: en").unwrap();
```

Comment:
It would ideally be good to parameterise the language used via configuration/command line arguments, in the future, rather than hard-coding it.

Reply:
Agreed. I did think about doing that, but since config.toml is entirely consumed by the IRC Server object and there's no CLI option parsing yet, I decided to be lazy...

Reply:
It appears from HTTP docs to be a hint/preference to the server? so "force" seems a bit strong... Also commit message could describe the problem this is solving (still curious), but otherwise happy to merge.